assets/sass/main.scss

```diff
@@ -672,9 +673,9 @@
             }
 
             thead {
-                background-color: #555555;
+                background-color: _palette(fg-bold);
```

Comment:
Using fg color for bg?

Reply:
I tried to refactor the colors so there are no more differences. I added the color `bg-light` for `#555555`.